tools/spatyper/spatyper.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="spatyper" name="Generate spa type identification" version="@VERSION@+galaxy0" profile="21.05">
+    <macros>
+        <token name="@VERSION@">0.3.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">spatyper</requirement>
+        <requirement type="package" version="3.4.3">python</requirement>
```

Comment:
Is it the case that `spatyper` requires exactly Python 3.4? Is the correct version of Python not already installed in the `spatyper` conda environment? If the conda environment already includes a version of Python that works with spatyper, then there's no need to specify an exact version of Python here.

Reply:
This was a bit of a longshot attempt at fixing the "Combine chunked test results" for the CI. I can definitely clear this line